### PR TITLE
Adding recipe for CADD-scripts.

### PR DESCRIPTION
**Pull request by holtgrewe**
Adding cadd-scripts v1.6

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Review by visze — CHANGES_REQUESTED** [review]
I think there should also be a wrapper for the script `install.sh`

**Comment by holtgrewe** [comment]
@visze I added a second wrapper `cadd-install.sh` What do you think?

File: recipes/cadd-scripts/cadd-install-wrapper.sh [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Wrapper for CADD.sh
+set -eu -o pipefail
+
+CADD=$PACKAGE_HOME \
+    $PACKAGE_HOME/CADD.sh "$@"
```
Comment (visze) [inline recipes/cadd-scripts/cadd-install-wrapper.sh:5]: should this be cadd-install? Otherweise the cadd-installwrapper looks the same as the cadd-wrapper

Comment (visze) [reply recipes/cadd-scripts/cadd-install-wrapper.sh:5]: or `install.sh`?

File: recipes/cadd-scripts/build.sh [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+# Copy files into $PACKAGE_HOME
+cp -r . $PACKAGE_HOME/.
+
+# Patch CADD.sh
+sed -i -e 's/^export CADD.*/export CADD=${CADD-$(dirname "$SCRIPT")}/' $PACKAGE_HOME/CADD.sh
+
+# Create wrappers
+SOURCE_FILE=$RECIPE_DIR/cadd-wrapper.sh
+DEST_FILE_CADD=$PACKAGE_HOME/cadd-wrapper.sh
+DEST_FILE_INST=$PACKAGE_HOME/cadd-wrapper.sh
```
Comment (visze) [inline recipes/cadd-scripts/build.sh:20]: `cadd-install-wrapper.sh`?

**Review by visze — CHANGES_REQUESTED** [review]
I think there are still some issues.

File: recipes/cadd-scripts/cadd-install-wrapper.sh (hunk 1 above)
Comment (holtgrewe) [reply recipes/cadd-scripts/cadd-install-wrapper.sh:5]: `install.sh`

File: recipes/cadd-scripts/build.sh (hunk 2 above)
Comment (holtgrewe) [reply recipes/cadd-scripts/build.sh:20]: Yes, sorry.

**Comment by holtgrewe** [comment]
I as kind of copy and paste failure agree ;)

**Review by visze — APPROVED** [review]
LGTM now

**Comment by holtgrewe** [comment]
@BiocondaBot please merge

**Pull request merged**